internal/servers/controller/handlers/credentialstores/credentialstore_service_test.go

```diff
@@ -552,13 +513,13 @@ func TestCreate(t *testing.T) {
 				got.Uri, tc.res.Uri = "", ""
 				got.Item.Id, tc.res.Item.Id = "", ""
 				got.Item.CreatedTime, got.Item.UpdatedTime, tc.res.Item.CreatedTime, tc.res.Item.UpdatedTime = nil, nil, nil, nil
-				if _, ok := got.Item.Attributes.Fields["token_hmac"]; ok {
-					assert.NotEqual(tc.req.Item.Attributes.Fields["token"], got.Item.Attributes.Fields["token_hmac"])
-					got.Item.Attributes.Fields["token_hmac"] = structpb.NewStringValue("<hmac>")
+				if got.Item.GetVaultCredentialStoreAttributes().TokenHmac != "" {
+					assert.NotEqual(tc.req.Item.GetVaultCredentialStoreAttributes().Token, got.Item.GetVaultCredentialStoreAttributes().TokenHmac)
+					got.Item.GetVaultCredentialStoreAttributes().TokenHmac = "<hmac>"
```

Comment:
Minor comment, but instead of overriding this, we can just ignore it in the comparison on line 525, which I think is slightly nicer: https://github.com/hashicorp/boundary/blob/517a3b3c127d223cc1d409ac5309dac1e353ce92/internal/servers/controller/handlers/authmethods/oidc_test.go#L1013-L1041.